include/fmt/ranges.h

```diff
@@ -128,24 +128,46 @@ struct is_tuple_like {
 };
 }  // namespace internal
 
-#if FMT_HAS_FEATURE(__cpp_lib_integer_sequence) || FMT_MSC_VER >= 1900
-# define FMT_USE_INTEGER_SEQUENCE 1
+// Check for integer_sequence
+#if defined(__cpp_lib_integer_sequence) || FMT_MSC_VER >= 1910
+template <typename T, T... N>
+using integer_sequence = std::integer_sequence<T, N...>;
+template <std::size_t... N>
+using index_sequence = std::index_sequence<N...>;
+template <std::size_t N>
+using make_index_sequence = std::make_index_sequence<N>;
 #else
-# define FMT_USE_INTEGER_SEQUENCE 0
+template <typename T, T... N>
+struct integer_sequence {
+    typedef T value_type;
+
+    static FMT_CONSTEXPR std::size_t size() {
+        return sizeof...(N);
+    }
```

Comment:
Please use 2-space indent for consistency (https://github.com/fmtlib/fmt/blob/master/CONTRIBUTING.rst).